BitmarkRegistry/Controllers/Main/Properties/BitmarkDetailViewController.swift

```diff
@@ -145,6 +147,9 @@ class BitmarkDetailViewController: UIViewController {
           guard let downloadedFileURL = downloadedFileURL else { return }
           let shareVC = UIActivityViewController(activityItems: [downloadedFileURL], applicationActivities: [])
           self.present(shareVC, animated: true)
+          self.assetR.updateAssetFilePath(downloadedFileURL.path, completion: { [weak self] in
```

Comment:
It's really weird that we need to wait for db writing before showing the image thumbnail.
Could we treat `AssetR` as a pure model, and update db after processing the data?

Reply:
It's ok, but I think about making the UI data all come based on assetR data.

I also intent to replacing this with making Realm listener of assetR record; so pending assetR can real-time update when changing to confirmed (example issued date ...)

What do you think?

Reply:
👍